api/controllers/articles.controllers.js

```diff
@@ -32,4 +32,17 @@ exports.getCommentsByArticleId = (req, res, next) => {
             res.status(200).send({ comments })
         })
         .catch(next)
+}
+
+exports.patchArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    const { inc_votes } = req.body
+    const promises = [checkCategoryExists('articles', 'article_id', article_id), updateArticleById(article_id, inc_votes)]
```

Comment:
This deffinitely works however you may not have to check if a category exists here since if we try an update an article that doesn't exist PSQL should handle the promise.reject